docs/getting_started/installation_guide/platforms/hopsworks-installer.rst

```diff
@@ -43,7 +43,7 @@ On your server/VM, run the following bash commands from your user account with s
 Setting up a Cluster for Installation
 -----------------------------------------------------------------
 
-You need to identify a set of servers/VMs and create the same user account with sudo privileges on all the servers. You should identify one server as the head server from which you will perform the installation. You need to configure password-less SSH Access from the Head node to Worker nodes. First, on the head node, you should create an openssh keypair without a password:
+You need to identify a set of servers/VMs and create the same user account with sudo privileges on all the servers. You should identify one server as the head server from which you will perform the installation. You need to ensure that the nodes can communicate with each other on every port and to configure password-less SSH Access from the Head node to Worker nodes. First, on the head node, you should create an openssh keypair without a password:
```

Comment:
We should add a comment after this:
The set of ports that need to be opened is available here.
(with a hyperlink to the doc with the list of open ports)

Reply:
> The set of ports that need to be opened is available here.

https://github.com/logicalclocks/hops-docs/pull/196